### PR TITLE
header js changes as per discussion

### PR DESCRIPTION
**Pull request by Ashish-srij**
@shishirphadke I have made the changes on header js as per discussion with @kuldev Please review and merge.

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
 import pnud from '../../../../assets/images/undp-logo-blue.svg';
+import { getData } from '../../../../assets/js/navigation-data';
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx:3]: @Ashish-srij Change the function name to `getMegaMenu`.

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
 import pnud from '../../../../assets/images/undp-logo-blue.svg';
+import { getData } from '../../../../assets/js/navigation-data';
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx:3]: @Ashish-srij Change the function name to `getMegaMenu`.

File: stories/assets/js/navigation.js [hunk 3]
```diff
@@ -125,18 +118,18 @@ export const navigationInitialize = (locale) => {
...
 
-  $('.mob-lang-switcher').click((ev) => {
+  $('.mob-lang-switcher').on('click', function(ev) {
     ev.preventDefault();
     $('.mob-sub-lang').addClass('show');
     $('.mobile-links').addClass('hide');
   });
 
-  $('.back-nav').click(() => {
+  $('.back-nav').on('click',() => {
```
Comment (shishirphadke) [inline stories/assets/js/navigation.js:127]: @Ashish-srij Can you add `function()` keyword here to maintain consistency with the code on line #121 and #132.

**Review by shishirphadke — APPROVED** [review]

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx (hunk 2 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx]: @shishirphadke Done.

**Review by shishirphadke — APPROVED** [review]

**Pull request merged**